config/pipelines/high_throughput_scrna_core_cdna_prep.yml

```diff
@@ -23,5 +23,6 @@ scRNA Core cDNA Prep GEM-X 5p GEMs Input:
     request_type_key: limber_scrna_core_cdna_prep_gem_x_5p_gems_input
   relationships:
     LRC GEM-X 5p GEMs Input: LRC GEM-X 5p cDNA PCR
-    LRC GEM-X 5p GEMs Input CITE: LRC GEM-X 5p cDNA PCR
+    LRC GEM-X 5p GEMs Input CITE: LRC GEM-X 5p cDNA PCR CITE
+    LRC GEM-X 5p cDNA PCR CITE: LRC GEM-X 5p cDNA PCR XP
```

Comment:
I think this branching would be clearer as a separate pipeline configuration - but, if it works fine like this (i.e. Int Suite test passes), then I think let's leave it, because Seena's story will be modifying it anyway.